src/main/resources/com/cloudbees/jenkins/plugins/bitbucket/PullRequestNamingTrait/help-namingExcludeRegex.html

```diff
@@ -0,0 +1,10 @@
+<div>
+    A <a href="https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html">Java regular expression</a> to
+    exclude specific pull request branch names from being affected by the selected naming strategy. Names that match
+    the supplied regular expression will not be changed (their original branch names will be used).
+    <p>Example: use <code>^release/.*|^hotfix/.*</code> to exclude all <code>release/*</code> and <code>hotfix/*</code>
+    branches.<br>Leave empty to apply the selected naming strategy to all pull requests.</p>
+    <strong>NOTE:</strong> Pull request discovery strategy classifiers (<code>merge</code> or <code>head</code>) will
+    still be appended (or prepended, depending on the naming strategy) if multiple pull request "discovery strategies"
+    are active at the same time. Even for branches matching the specified pattern.
```

Comment:
This means that if more than one PR is created from a branch that matches the exclusion, they will collide, correct? 
The would be very unlikely within a single repo, but for forks... It is very possible that two users could pick the same branch name in their own fork and then each create a pull request. 

Would you consider removing this part of the feature from this PR?  I'm not saying you have to, I'm asking if you consider this a must-have part of this change. 

Removing this would simplify this PR, and then you could come back and submit another PR with this additional functionality.